app/gui/view/graph-editor/src/lib.rs

```diff
@@ -94,6 +94,9 @@ pub mod prelude {
 
 const SNAP_DISTANCE_THRESHOLD: f32 = 10.0;
 const VIZ_PREVIEW_MODE_TOGGLE_TIME_MS: f32 = 300.0;
+/// Number of frames we expect to pass during the
+/// `VIZ_PREVIEW_MODE_TOGGLE_TIME_MS` interval. Assumes 60fps.  
+const VIZ_PREVIEW_MODE_TOGGLE_FRAMES: i32 = (VIZ_PREVIEW_MODE_TOGGLE_TIME_MS / 16.0) as i32;
```

Comment:
If we assume 60 FPS I would rather put here 1000.0/60.0. Or make assumed FPS another constant